sdk/newrelicobservability/Azure.ResourceManager.NewRelicObservability/api/Azure.ResourceManager.NewRelicObservability.netstandard2.0.cs

```diff
@@ -55,566 +55,566 @@ public partial class NewRelicMonitorResourceCollection : Azure.ResourceManager.A
     public partial class NewRelicMonitorResourceData : Azure.ResourceManager.Models.TrackedResourceData
     {
         public NewRelicMonitorResourceData(Azure.Core.AzureLocation location) : base (default(Azure.Core.AzureLocation)) { }
-        public Azure.ResourceManager.NewRelicObservability.Models.AccountCreationSource? AccountCreationSource { get { throw null; } set { } }
+        public Azure.ResourceManager.NewRelicObservability.Models.NewRelicObservabilityAccountCreationSource? AccountCreationSource { get { throw null; } set { } }
         public Azure.ResourceManager.Models.ManagedServiceIdentity Identity { get { throw null; } set { } }
         public Azure.ResourceManager.NewRelicObservability.Models.LiftrResourceCategory? LiftrResourceCategory { get { throw null; } }
```

Comment:
I'm not sure if this name `LiftrResourceCategory` will be used by other Liftr services.

Reply:
Yes, it is used by other RPs, updated.